jobs/germany-crowd-forecast.sh

```diff
@@ -6,8 +6,20 @@ docker exec -w /home/rstudio/covid-german-forecasts forecast git pull -Xours
 # update dependencies
 docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript -e 'devtools::install_github("epiforecasts/covid-german-forecasts")'
 
+# create crowd forecast app submission data
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript human-forecast/update-submissions.R
+
+# Redeploy forecast app (to update submission date to next week)
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript human-forecast/redeploy-forecast-app.R
+
 # simulate cases from Rt crowd forecast
 docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript rt-crowd-forecast/simulate-cases.R
 
+# Redeploy Rt forecast app (to update submission date to next week)
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript rt-crowd-forecast/redeploy-rt-crowd-forecast-app.R
+
+# rerun evaluation / leaderboard
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript evaluation/forecast-evaluation.R
```

Comment:
Again repeated and would be nice in a sub bash file.

Reply:
I added a sub bash file for updating the repo dependencies. would you like to have a second bash file for the evaluation as well?